src/test/java/dk/kb/present/solr/EmbeddedSolrFieldAnalyseTest.java

```diff
@@ -127,15 +127,27 @@ public void testTitleStrict() throws SolrServerException, IOException {
 
 	@Test
 	public void testAnalysis() throws SolrServerException, IOException {
+		addSimpleFieldTestDocuments();
 		SolrQuery solrQuery = new SolrQuery();
 		solrQuery.setQuery("id:1");
 		solrQuery.setRows(10);
 		QueryResponse rsp = embeddedServer.query(solrQuery, METHOD.POST);
 		//System.out.println(rsp.getResults().get(0).toString());
 	}
 	
-
-	private long getCreatorNameStrictResultsForQuery(String query) throws Exception{	    
+	@Test
+	public void testSynonymTest() throws SolrServerException, IOException {
+		addSynonymFieldTestDocuments();
+		assertEquals(1, getTitleSynonymQuery("tvavis"));
+		assertEquals(1, getTitleSynonymQuery("\"tvavis\"")); // "tvavis" in quotes
+		assertEquals(1, getTitleSynonymQuery("tv-avisen")); //"tv-avisen" in quotes
+		assertEquals(1, getTitleSynonymQuery("\"tv-avisen\""));
```

Comment:
Eclipse has been using tabs for indent, both in new code in this test class and for a lot of the old code in this class. Please fix this in the file, so that you @thomasegense will retain git credit for the code.

Also, please tell Eclipse to _never_ use tabs for indent for Java code.

Reply:
configured eclipse again... And reformated file.